Confusion about optional dependencies

In a new Symfony project, install Maker and run this:

```bash
$ php bin/console make:functional-test


 [ERROR] Missing packages: to use the make:functional-test command, run:

         composer require browser-kit css-selector panther --dev
```

`panther` is optional, but it's displayed nevertheless to install it.

However, if I install manually both `browser-kit` and `css-selector` (but not `panther`) the error message disappear and everything works.

So, optional dependencies are indeed optional ... but it's confusing to add them to the command that installs things to fix problems.

--- a/src/Maker/MakeFunctionalTest.php
+++ b/src/Maker/MakeFunctionalTest.php
@@ -84,11 +84,5 @@ class MakeFunctionalTest extends AbstractMaker
             true,
             true
         );
-        $dependencies->addClassDependency(
-            PantherTestCaseTrait::class,
-            'panther',
-            false,
-            true
-        );
     }
 }

